qadence/types.py

```diff
@@ -459,3 +459,28 @@ class ExperimentTrackingTool(StrEnum):
 
 
 LoggablePlotFunction = Callable[[Module, int], tuple[str, Figure]]
+
+
+class NoiseProtocolType(StrEnum):
```

Comment:
Would it make sense to merge this two enums somehow ? So that you could do smtg like `NoiseType.ANALOG.DEPOLARIZING` ?

Reply:
Made a dataclass `NoiseProtocol` with different levels of noises.